web-test-runner.config.js

```diff
@@ -0,0 +1,55 @@
+import rollupCommonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import { fromRollup, rollupAdapter } from '@web/dev-server-rollup';
+import { cemAnalyzerPlugin } from './wds/cem-analyzer-plugin.js';
+import { esbuildBundlePlugin } from './wds/esbuild-bundle-plugin.js';
+
+const commonjs = fromRollup(rollupCommonjs);
+
+function commonJsIdentifiers (ids) {
+  return ids.map((id) => `**/node_modules/${id}/**/*`);
+}
+
+export default {
+  files: ['test/**/*.test.*', 'src/components/**/*.test.*'],
+  nodeResolve: true,
+  mimeTypes: {
+    '**/*.md': 'js',
+    '**/*.json': 'js',
+    '.**/*.json': 'js',
+  },
+  testFramework: {
+    config: {
+      ui: 'bdd',
+      timeout: '5000',
+    },
+  },
+  testRunnerHtml: (testFramework) => `
+    <html>
+      <body>
+        <link rel="stylesheet" href="src/styles/default-theme.css" >
+        <script type="module" src="${testFramework}"></script>
+      </body>
+    </html>
+  `,
+  plugins: [
+    cemAnalyzerPlugin,
```

Comment:
* Why do we need the CEM analyzer plugin in the test runner?
  * I guess it allows us to import stories and some of them do import MD/JSON files, right?

Reply:
`**/*.json` mimeType + `cemAnalyzerPlugin` are useful because `makeStory` imports the CEM. 

But it's true that the part of `makeStory` that requires CEM is only useful for storybook and not for tests.

Do you think we should optimize this ? (I do see how, but I did not want to modify the `makeStory` function unless I was told to do so, considering how many changes we're already introducing :smile: )